components/snowflake/sources/new-table/new-table.mjs

```diff
@@ -0,0 +1,50 @@
+import common from "../common.mjs";
+import { v4 as uuidv4 } from "uuid";
+
+export default {
+  ...common,
+  type: "source",
+  key: "snowflake-new-table",
+  name: "New Table",
+  description: "Emit new event when a table is created",
```

Comment:
```suggestion
  description: "Emit a new event when a table is created",
```

Reply:
Changed "Emit new event" to "Emit a new event" to follow Google's Developer docs style guide (use articles)